### ISSUE TITLE
Add ability to configure selector for nav buttons

### ISSUE THREAD

**Issue by benjamw** [fancyapps/fancyBox#912]
In my instance, I don't want the nav buttons to cover the entire left and right quarter of the item as there are things inside that need to be clickable. I therefore changed my templates to the following:

```
tpl: {
    next: '<div class="fancybox-nav fancybox-next"><a title="Next" href="javascript:;"></a></div>',
    prev: '<div class="fancybox-nav fancybox-prev"><a title="Prev" href="javascript:;"></a></div>'
},
```

Because of this change, the selector that binds the click event to the nav buttons is no longer valid.
I changed the code to the following in the proper places:

```
$(current.tpl.prev).appendTo(F.outer).find('a').addBack('a').bind('click.fb', F.prev);
```

and that worked, but I feel dirty editing the core.

So basically, there needs to be a selector that is configurable that can be used to select the relevant part of the template that should be used as the click target.

**Comment by fancyapps** [comment]
Hi, please, upgrade to v3.
